client/src/extension.ts

```diff
@@ -83,12 +83,18 @@ export function activate(context: vscode.ExtensionContext) {
           endpoint,
           new AzureKeyCredential(azureApiKey)
         );
-        //result variable will be used in future implementation
         const result = await client.getChatCompletions(
           deploymentId,
           messages,
           parameters
         );
+        const editor = vscode.window.activeTextEditor;

```

Comment:
What if there is no active document open? Should the extension create one?

Reply:
Hi, the extension throws an error if there is no active document open. I can incorporate functionality that create a file to prevent an error being thrown, though.

Reply:
If this throws, the extension should definitely catch that exception and either show a notification or create a document with a new name (could ask user for input if document does not exist to make sure an existing document does not get overwritten).